src/common/settings.cc

```diff
@@ -109,12 +111,48 @@ string Settings::SettingsImpl::getip () const
 }
 // }}}
 // Get specializations {{{
-template<typename T> T Settings::SettingsImpl::get (string& str) {
-   return pt.get<T> (str.c_str());
+//template<typename T> T Settings::SettingsImpl::get (string& str) {
```

Comment:
Why are they commented out? Do we need it later? It would be better to just remove the code if we don't need it.